### ISSUE TITLE
[ALERT BOX]: Need to revert the hard-coded role value for screen reader usability

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/veteran-facing-services-tools#362]
**Describe the bug**

PR #309 added a `role="alert"` value to the Alertbox component. The VSP frontend tools team is reverting this PR because it is causing all screen readers to interrupt normal document readout to announce Alertbox values. There should be a way to add a role when instances of the Alertbox component are used, but I'd like to not use a hard-coded value.

/cc @zurbergram for visibility

**Expected behavior**

- [ ] Alert boxes should not be announced immediately unless specifically flagged for `role="alert"`
